.gitignore

```diff
@@ -50,3 +50,8 @@ web.config
 *.v11.suo
 *~
 config/ampache.cfg.php.backup
+
```

Comment:
put this just in the file as normal
![image](https://user-images.githubusercontent.com/1305249/147995500-9ab3af1b-d0fc-4e7b-9431-9b8d550cd2b7.png)